src/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+// import reportWebVitals from './reportWebVitals';
```

Comment:
- [ ]  As I mentioned above make your code clean, kindly remove commented codes from your code!

Reply:
Thanks a lot, I'll do that 😄